.github/workflows/preview.yml

```diff
@@ -22,29 +22,30 @@ jobs:
         with:
           version: '1.10'
 
+      - name: Instantiate Julia environment
```

Comment:
We were instantiating environment in quarto file itself so we will do it this way then that should be removed I guess! See https://github.com/TuringLang/docs/blob/1f797b78268685c853efb5a32c3c71744152e47b/tutorials/00-introduction/index.qmd#L8C1-L13C4